website/content/en/preview/AWS/provisioning.md

```diff
@@ -6,6 +6,36 @@ description: >
   Learn AWS-specific Karpenter provisioning settings
 ---
 
+Provisioner settings specific to Karpenter for the AWS cloud provider are described here.
+The following example shows optional and required settings for a Karpenter provisioner for AWS:
+
+
+```yaml
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:                      
+    - key: karpenter.sh/capacity-type         # optional, set to on-demand by default
+      operator: In
+      values: ["spot"]
+  limits:   
+    resources:
+      cpu: 1000                               # optional, recommended to limit total cluster CPU
+      memory: 1000Gi                          # optional, recommended to limit total cluster memory
+  provider:

```

Comment:
we need to document all the parameters in this struct https://github.com/aws/karpenter/blob/main/pkg/cloudprovider/aws/apis/v1alpha1/provider.go#L29

Reply:
I added all the parameters in that struct to the example. All are described below. Let me know if AMIFamily, InstanceProfile, and Tags examples are okay.